# All lines beginning with `# DP:' are a description of the patch.
# DP: Description: Minor fixes in te_IN
# DP: Dpatch author: Denis Barbier <barbier@debian.org>
# DP: Patch author: Denis Barbier
# DP: Upstream status: not submitted
# DP: Date: 2006-01-19

Index: glibc-2.3.5/localedata/locales/te_IN
===================================================================
--- glibc-2.3.5.orig/localedata/locales/te_IN
+++ glibc-2.3.5/localedata/locales/te_IN
@@ -150,7 +150,7 @@
 
 %
 % Syntax on Unicode Regular Expression Guidelines TR#18
-% ^[yY\u0C15].*
+% ^[nN\u0C15].*
 noexpr      "<U005E><U005B><U006E><U004E><U0C15><U005D><U002E><U002A>"
 
 %
@@ -187,23 +187,26 @@
 % generated by IBM Basic CountryPack Transformer.
 
 % postal_fmt    "%n%N%a%N%d%N%f%N%r%t%e%N%h%t%b%N%s%t%l%N%T%t%z%N%S%N%c"
-postal_fmt      "<U0025><U006E><U0025><U004E>/
-<U0025><U0061><U0025><U004E>/
-<U0025><U0064><U0025><U004E>/
-<U0025><U0066><U0025><U004E>/
-<U0025><U0072><U0025><U0074><U0025><U0065><U0025><U004E>/
-<U0025><U0068><U0025><U0074><U0025><U0062><U0025><U004E>/
-<U0025><U0073><U0025><U0074><U0025><U006C><U0025><U004E>/
-<U0025><U0054><U0025><U0074><U0025><U007A><U0025><U004E>/
-<U0025><U0053><U0025><U004E>/
-<U0025><U0063>"
+% postal_fmt      "<U0025><U006E><U0025><U004E>/
+% <U0025><U0061><U0025><U004E>/
+% <U0025><U0064><U0025><U004E>/
+% <U0025><U0066><U0025><U004E>/
+% <U0025><U0072><U0025><U0074><U0025><U0065><U0025><U004E>/
+% <U0025><U0068><U0025><U0074><U0025><U0062><U0025><U004E>/
+% <U0025><U0073><U0025><U0074><U0025><U006C><U0025><U004E>/
+% <U0025><U0054><U0025><U0074><U0025><U007A><U0025><U004E>/
+% <U0025><U0053><U0025><U004E>/
+% <U0025><U0063>"
+%  Localedef chokes on above definition, so keep the old one even if
+%  it is wrong.
+postal_fmt  "<U0025><U007A><U0025><U0063><U0025><U0054><U0025><U0073><U0025><U0062><U0025><U0065><U0025><U0072>"
 country_name    "<U0C2D><U0C3E><U0C30><U0C24><U0020><U0C26><U0C47><U0C36><U0C02>"
 country_ab2     "<U0049><U004E>"
 country_ab3     "<U0049><U004E><U0044>"
 country_num     356
 lang_name       "<U0C24><U0C46><U0C32><U0C41><U0C17><U0C41>"
-lang_ab         "te"
-lang_term       "tel"
+lang_ab         "<U0074><U0065>"
+lang_term       "<U0074><U0065><U006C>"
 END LC_ADDRESS
 
 
